src/akkudoktoreos/class_optimize.py

```diff
@@ -645,5 +645,8 @@ def optimierung_ems(
             "eauto_obj": ems.eauto.to_dict(),
             "start_solution": start_solution,
             "spuelstart": spuelstart_int,
-            # "simulation_data": o,
+            "simulation_data": o,
+            "extra_data": extra_data,
```

Comment:
If the API changes, please extend OptimizeResponse as well (extra_data). Do we need simulation_data? Atm it's just a copy of results.

Reply:
you are right, no we do not.